packages/react-native-editor/__device-tests__/gutenberg-editor-cover.test.js

```diff
@@ -33,17 +38,20 @@ describe( 'Gutenberg Editor Cover Block test', () => {
 		await expect( editorPage.getBlockList() ).resolves.toBe( true );
 	} );
 
-	it( 'should convert height properly', async () => {
+	it( 'should displayed properly and have properly converted height (ios only)', async () => {
 		await editorPage.setHtmlContent( testData.coverHeightWithRemUnit );
 
 		const coverBlock = await editorPage.getBlockAtPosition(
 			coverBlockName
 		);
-		const { height } = await coverBlock.getSize();
-		// Height is set to 20rem, where 1rem is 16.
-		// There is also block's vertical padding equal 32.
-		// Finally, the total height should be 20 * 16 + 32 = 352
-		expect( height ).toBe( 352 );
+
+		if ( ! isAndroid() ) {
+			const { height } = await coverBlock.getSize();
+			// Height is set to 20rem, where 1rem is 16.
+			// There is also block's vertical padding equal 32.
+			// Finally, the total height should be 20 * 16 + 32 = 352
+			expect( height ).toBe( 352 );
+		}
```

Comment:
👋 @lukewalczak, can we add a comment on why do we avoid the check on Android? I know we're working in parallel to make it work on Android but to help someone that doesn't follow this work too closely, let's add a comment for the time being.

Reply:
done 👍🏼